src/app/people/people.ts

```diff
@@ -110,6 +112,14 @@ export class PeopleComponent {
       this.sortedItems = this.items.sort((a, b) => {
         return a.created_at! < b.created_at! ? 1 : -1;
       });
+    } else if (sorting === 'interesting-asc') {
+      this.sortedItems = this.items.sort((a, b) => {
```

Comment:
I see here why you wanted the .interesting field, hm.. we probably need this logic to change and use the MetricService.